jekyll/_cci2_ja/introduction-to-the-circleci-web-ui.adoc

```diff
@@ -0,0 +1,181 @@
+---
+
+version:
+- クラウド
+- Server v2.x
+- Server v3.x
+---
+= CircleCI Web UI について
```

Comment:
```suggestion
= CircleCI Web アプリについて
```

Reply:
I‘ll do this next time 🙏  (The content should be fixed as well, and Crowdin will cover this next time.)